Packs/GCP-IAM/Integrations/GCP-IAM/GCP-IAM.py

```diff
@@ -0,0 +1,1939 @@
+        args (dict): Request function arguments.
+
+    Returns:
+        str: Continuation token
+
+    """
+    offset = limit * (page - 1)
+    response = client_request(limit=offset, **args)
+    return response.get('nextPageToken')
+
+
+def validate_pagination_arguments(limit: int, page: int) -> None:
+    """
+    Validate pagination arguments values.
+    Args:
+        limit (int): Number of elements to retrieve.
+        page (int): Page number.
+
+    """
+    if page < 1 or limit < 1:
+        raise Exception('Page and limit arguments must be greater than 0.')
+
+
+def update_time_format(data: dict, keys: list) -> None:
+    """
+    Update dictionary time values to appropriate XSOAR system time format.
+    Args:
+        data (dict): Information to update.
+        keys (list): Keys to update.
+
+
+    """
+    if not isinstance(data, list):
+        data = [data]
+
+    for item in data:
+        for key in keys:
+            if key in item:
+                item[key] = arg_to_datetime(item[key]).isoformat()
+
+    return data
+
+
+def generate_iam_policy_command_output(response: dict, resource_name: str) -> CommandResults:
+    """
+    Generate command output for iam-policy commands.
+    Args:
+        response (dict): API response from XSOAR.
+        resource_name (str): The resource for which the policy is being specified.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+
+    """
+    outputs = copy.deepcopy(response)
+    outputs['name'] = resource_name
+
+    readable_output = tableToMarkdown(
+        f'{resource_name} IAM policy information:',
+        outputs.get('bindings'),
+        headers=['role', 'members'],
+        headerTransform=pascalToSpace
+    )
+
+    command_results = CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='GCP.IAM.Policy',
+        outputs_key_field='name',
+        outputs=outputs,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def remove_members_from_policy(role: str, iam_policy: list, members: list, command_name: str) -> list:
+    """
+    Remove members from policy role.
+    Args:
+        role (str): The name of policy role.
+        iam_policy (list): IAM policies.
+        members (list): Members to remove from policy.
+        command_name (str): An alternative  command that will be displayed to the user in case of an error.
+
+    Returns:
+        list: Updated policies.
+
+    """
+    role_found = False
+    for policy in iam_policy:
+        if policy.get('role') == role:
+            for member in members:
+                try:
+                    policy['members'].remove(member)
+                except ValueError:
+                    raise Exception(f'The member {member} is not part of the project IAM policies members.'
+                                    f'If you wish to add a new policy, consider using the {command_name} command.')
```

Comment:
@dantavori Can you please elaborate on the question?
The command responsibility is to remove members from IAM policy. 
If the user mentioned some member that is not a part of the policy - the command will raise an exception.
I can change the command so that it does not raise an exception in such a case, but this is the way your team asked to handle such a case.